### ISSUE TITLE
Illegal reflective access on JDK11 w/ classgraph 4.8.126

### ISSUE THREAD

**Issue by josephlbarnett** [classgraph/classgraph#579]
Using classgraph 4.8.126 on JDK11, we're seeing this in logs:
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by nonapi.io.github.classgraph.reflection.StandardReflectionDriver$1 (file:/....../server-1.0-SNAPSHOT-shaded.jar) to method sun.management.RuntimeImpl.getInputArguments()
WARNING: Please consider reporting this to the maintainers of nonapi.io.github.classgraph.reflection.StandardReflectionDriver$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

The warning does not show up with classgraph 4.8.116 on JDK11, or with classgraph 4.8.126 on JDK17.  We are putting all our dependencies on the classpath, not the modulepath, fwiw, and are seeing this both with a shaded jar as seen above, and when running with actual jar deps on the classpath.

Appears this is triggered by executing the following line in `ModulePathInfo.java`:
```
final List<String> commandlineArguments = runtimeMXBean == null ? null
                : (List<String>) ReflectionUtils.invokeMethod(/* throwException = */ false, runtimeMXBean,
                        "getInputArguments");
```

**Comment by lukehutch** [comment]
@josephlbarnett Thanks for the report. I checked some changes into git that should hopefully solve this problem. Can you please test the latest git version?

**Comment by josephlbarnett** [comment]
latest git looks good in my limited testing on jdk11 and 17

**Comment by lukehutch** [comment]
@josephlbarnett That level of testing should be sufficient to validate the fixed approach to reflection.

Thanks for testing -- fixed and released in 4.8.127.
